includes/blocks/class-sensei-course-navigation-block.php

```diff
@@ -276,6 +276,13 @@ private function render_svg_icon_library() {
 			<symbol id="sensei-lesson-status-completed" viewBox="0 0 24 24">
         <path fill-rule="evenodd" d="M12 20a8 8 0 100-16 8 8 0 000 16zm-1.024-3.949l5.764-7.753-.802-.596-5.466 7.351-2.942-2.187-.596.802 3.342 2.486.402.298.298-.401z"/>
 			</symbol>
+			<symbol id="sensei-contact-teacher-success" viewBox="0 0 42 42" fill="none">
+				<circle cx="21" cy="21" r="20.25" stroke="#30968B" stroke-width="1.5"/>
+				<path d="M27.1135 15.1507L18.8804 26.2233L14.1064 22.6735" stroke="#30968B" stroke-width="2"/>
```

Comment:
As a general approach, we try to leave colors out of SVGs, and style them from CSS. Also it would make life easier later to use a standard `viewBox="0 0 24 24"`, and set the desired size from CSS.

Reply:
Sure! I removed the colors and set them via CSS. 91d185a1d8f1225608f3630b6cd5611447e123f5

It's a bit weird with shadow elements when we do `<use xlink:href="..." />`. Could not target individual `<circle>` and `<path>` elements of the icon. Had to do `svg * { stroke: green; }` to make it work.

Reply:
> Also it would make life easier later to use a standard `viewBox="0 0 24 24"`, and set the desired size from CSS.

Simply setting the desired viewbox size does not work. The coordinates of the `path` elements expect the viewbox size to be 42. I had to use third party tool to resize the whole svg. 2d8bb123341b0be5172b800be08e307898f256dd

> In Gutenberg, and also for most of our icons, usually everything is converted into a single path (or a few if a single one is not possible), including strokes. Keeps things simpler.

I managed it to have only `path` elements. Still doesn't work. The main issue is the `all:reset` that we do. It removes all the `stroke`, `stroke-width` values from svg `path` elements. You can set them manually with css (which we shouldn't), that works on Firefox, but on chrome the icons does not work at all. 

We need to solve this problem asap, otherwise it will accumulate lots of dept. Not in scope of this PR though.

Reply:
Built and exported one in Figma from the designs, this should work better:

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path fill-rule="evenodd" d="M12 23.143c6.154 0 11.143-4.989 11.143-11.143C23.143 5.846 18.154.857 12 .857 5.846.857.857 5.846.857 12c0 6.154 4.989 11.143 11.143 11.143zM12 24c6.627 0 12-5.373 12-12S18.627 0 12 0 0 5.373 0 12s5.373 12 12 12z"/>
  <path fill-rule="evenodd" d="M15.951 8.998l-5.045 6.786-3.187-2.37.682-.917 2.27 1.688 4.363-5.869.917.682z"/>
</svg>
```

Reply:
Doesn't look good. Not sure what's the problem, but I am getting a little distorted success icon with the svg that you exported. See below

#### Before 
<img width="520" alt="Screen Shot 2021-12-13 at 12 27 11 PM" src="https://user-images.githubusercontent.com/2578542/145777654-faeb88f2-4466-4d7c-a360-3bb1bfe241d5.png">

#### After
<img width="518" alt="Screen Shot 2021-12-13 at 12 26 49 PM" src="https://user-images.githubusercontent.com/2578542/145777661-0cf50da2-2a96-4a23-96fb-9650c0d0af37.png">

Reply:
I think that's the stroke CSS, need to remove that. (I converted the stroke to a path)
Looks like all our `<use>` SVGs are broken in Chrome. Added it to the scope of  #4482